cmd/commands/function.go

```diff
@@ -121,9 +135,10 @@ func FunctionCreate(fcTool *core.Client) *cobra.Command {
 	command.Flags().StringVar(&createFunctionOptions.Image, "image", "", "the name of the image to build; must be a writable `repository/image[:tag]` with credentials configured")
 	command.MarkFlagRequired("image")
 	command.Flags().StringVar(&createFunctionOptions.GitRepo, "git-repo", "", "the `URL` for a git repository hosting the function code")
-	command.MarkFlagRequired("git-repo")
 	command.Flags().StringVar(&createFunctionOptions.GitRevision, "git-revision", "master", "the git `ref-spec` of the function code to use")
-	command.Flags().StringVar(&createFunctionOptions.Handler, "handler", "", "the name of the `method or class` to invoke, depending on the invoker used")
+	command.Flags().StringVarP(&createFunctionOptions.LocalPath, "local", "l", "", "path to local source to build the image from")
+	command.Flag("local").NoOptDefVal = "."
```

Comment:
`--local` doesn't work with a space separated value then apparently:
`--local /tmp/foo` => wrong (thinks that's a 3rd arg)
`--local=/tmp/foo` => OK